src/spyglass/decoding/v1/waveform_features.py

```diff
@@ -0,0 +1,273 @@
+import os
+import shutil
+import uuid
+from pathlib import Path
+
+import datajoint as dj
+import numpy as np
+import pandas as pd
+import pynwb
+import spikeinterface as si
+
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.spikesorting.merge import SpikeSortingOutput
+from spyglass.utils.dj_mixin import SpyglassMixin
+
+schema = dj.schema("waveform_features")
+
+
+@schema
+class WaveformFeaturesParams(SpyglassMixin, dj.Manual):
+    """Defines the types of spike waveform features computed for a given spike
+    time."""
+
+    definition = """
+    waveform_features_param_name : varchar(80) # a name for this set of parameters
+    ---
+    waveform_features : BLOB  # a list of waveform features
+    waveform_features_params: BLOB # dictionary of dictionary of parameters for each waveform feature
+    waveform_extraction_params : BLOB # parameters for extracting waveforms
```

Comment:
It adds a lot to a table view to see the prefix `waveforms_` in front of each field, when it's implied by the table name. 

Three blobs seems like a lot for a table, when one blob could support a dict with each of the three embedded. For features that might be used to restrict a table, it's helpful to have separate fields, but I don't think that's the case for blobs